### PR TITLE
Fix requirements

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
This PR in response to #1244.

The `constraint` package was unable to be installed using pip3.  Error message is below.  Manual installation of the latest python constraint works fine. AFAIK, none of the scriptware in core-v-verif uses the python constraint package so it has been removed (commented out) of the requirements file for now.

```
Collecting constraint==0.4.1
  Using cached constraint-0.4.1.tar.gz (48 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [7 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-oia56g2q/constraint_b991aa459eea49aaad63010590661ea1/setup.py", line 143
          except OSError, ex:
                 ^^^^^^^^^^^
      SyntaxError: multiple exception types must be parenthesized
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

### CODE REVIEW

File: bin/README.md
Comment (silabs-robin) [inline bin/README.md:11]: This PR also deletes `RV32Zifencei_Extension_Instruction copy.xlsx`.
Can you confirm that this is intentional?

That ISA fencei vplan has _more_ info in it than the non-"copy" ISA fencei vplan.
But I agree that this PR makes the right move, in keeping the simpler ISA fencei vplan and deferring the fencei semantics details to core-specific vplans.

File: bin/requirements.txt
Comment (silabs-robin) [inline bin/requirements.txt:4]: I have momentarily seen an error about jinja2 myself recently, but then I had not configured my shell properly.
Since our IT setup is different, I don't anticipate that this change will affect us.

Is the usage of jinja2 within the python scripts still functioning after upgrading to major version 3 like this?
If yes, then I do not oppose the change.

File: docs/VerifStrat/source/quick_start.rst
Comment (zchamski) [inline docs/VerifStrat/source/quick_start.rst:43]: LGTM.  We should be systematic in using Python3 and pruning any possibility of using Python2 flavors.

File: bin/requirements.txt
Comment (zchamski) [reply bin/requirements.txt:4]: I'm checking the status of Jinja changes on the CVA6 side, will report the feedback here when available.

Comment (zchamski) [reply bin/requirements.txt:4]: > I'm checking the status of Jinja changes on the CVA6 side, will report the feedback here when available.

No adverse impact in CVA6 (already using `Jinja2 v3.x`, not using `constraint`) ==> LGTM.

File: bin/README.md
Comment (MikeOpenHWGroup) [reply bin/README.md:11]: Yes, this was intentional.  I assumed that including it was a fat-finger-issue.  Let me know if you wish to keep it.

File: docs/VerifStrat/source/quick_start.rst
Comment (MikeOpenHWGroup) [reply docs/VerifStrat/source/quick_start.rst:43]: Agreed!

**Comment by MikeOpenHWGroup** [comment]
Hi @JeanRochCoulon, can you have a look as well?  I need a Committer to review/approve before merging.  Thanks.

**Review by JeanRochCoulon — APPROVED** [review]
LGTM !

File: bin/README.md
Comment (silabs-robin) [reply bin/README.md:11]: I too assumed it was a fat-finger-issue. I tip in favor of purging it like this PR does.

However, even if "ISA-general" vplans are more superficial than "core-specific" vplans (which might include micro architecture and implementation-specific decicions), the Vol I spec _does_ say that _"A FENCE.I instruction ensures that
a subsequent instruction fetch on a RISC-V hart will see any previous data stores already visible to the same RISC-V hart"_.
So it _could_ be that the non-"copy" vplan is a bit too thin.
But that is a methodology/philosophy question that is a bit on the side of the Python fixing of this PR.

**Pull request merged**
